functional/durable-attestion-sanity-on-localhost/test.sh

```diff
@@ -60,7 +60,11 @@ rlJournalStart
         rlRun "limeWaitForAgentRegistration ${AGENT_ID}"
 
         # create refstat from fake binary_bios_measurements
-        rlRun "python3 /usr/share/keylime/scripts/create_mb_refstate /var/tmp/binary_bios_measurements mb_refstate.txt"
+        if command -v keylime-policy; then
+            rlRun "keylime-policy -e /var/tmp/binary_bios_measurements -o mb_refstate.txt"
```

Comment:
It should be `keylime-policy create measured-boot -e -e /var/tmp/binary_bios_measurements -o mb_refstate.txt` here.